src/check.js

```diff
@@ -0,0 +1,13 @@
+import Store from './storage.js';
+
+export default function updateStatus(checkbox) {
+  const { id } = checkbox;
+  let tasks = Store.getToDos();
+  tasks = tasks.map((task) => {
+    if (task.id.toString() === id) {
+      return { ...task, completed: !task.completed };
+    }
+    return task;
+  });
+  localStorage.setItem('tasks', JSON.stringify(tasks));
```

Comment:
- [OPTIONAL] For a task checked as completed, it would be nice if the checkbox remained checked even when the page is refreshed. 💭